src/config/dynamic-theme-fixes.config

```diff
@@ -9646,6 +9646,25 @@ INVERT
 
 ================================
 
+pomofocus.io
+
+INVERT
+img[alt="option icon"]
+img[alt="remove icon"]
+img[alt="clear icon"]
+img[alt="save icon"]
+img[alt="plus icon"]
+
+CSS
+.gLMzBo, .dAOjuP, .kfPydZ {
```

Comment:
What elements are these?

Reply:
My bad!, they were dynamic classnames. i've fixed them in d016f1099581f03aef6b16312f15eb010323a50e
They are meant to change the color of the website background 
![image](https://user-images.githubusercontent.com/60839576/128724145-6d6d5fb7-c543-421f-9dbc-fb42c9e5e6e3.png)

Reply:
The icon's are from the mini menu 
![image](https://user-images.githubusercontent.com/60839576/128724441-654bd917-8dee-4eab-aebd-3655c6175b9b.png)